src/atoms/hooks/index.ts

```diff
@@ -7,31 +7,25 @@ import { useReleaseNotesQuery } from './useReleaseNotesQuery';
 import { useSelect } from './useSelect';
 import { useSignalRMessages } from './useSignalRMessages';
 import { useOnScreenMultiple } from 'src/atoms/hooks/useOnScreenMultiple';
-import { useFeatureToggling } from 'src/deprecated/useFeatureToggling';
 import { useNotification } from 'src/organisms/TopBar/Notifications/NotificationProvider';
 import { useAuth } from 'src/providers/AuthProvider/AuthProvider';
-import { useReleaseNotes } from 'src/providers/ReleaseNotesProvider';
 import { useSideBar } from 'src/providers/SideBarProvider';
 import { useSnackbar } from 'src/providers/SnackbarProvider';
 import { useTableOfContents } from 'src/providers/TableOfContentsProvider';
-import { useTutorialSteps } from 'src/providers/TutorialStepsProvider';
```

Comment:
This is the old tutorial provider, which should stay deprecated I think, as the new one is completely different, and would be a "bigger" breaking change when updating to ACL 8.0

Reply:
Reverted the removal of it now